packages/block-library/src/template-part/edit/advanced-controls.js

```diff
@@ -73,8 +73,7 @@ export function TemplatePartAdvancedControls( {
 			{ isEntityAvailable && (
 				<>
 					<TextControl
-						// TODO: Switch to `true` (40px size) if possible
-						__next40pxDefaultSize={ false }
+						__next40pxDefaultSize
```

Comment:
Can be seen when you select a template part block (like "Header") in the site editor.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/70c98855-1944-498e-9e44-5c7701688568" alt="Template Part block inspector, before" width="258">|<img src="https://github.com/user-attachments/assets/7dc3ee8e-8ebb-45d7-87d3-077ae2031ba2" alt="Template Part block inspector, after" width="258">|